src/main/webapp/scripts/sys/isoFipsCountryMapScripts.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * CU Generic ISO-FIPS Country modification
+ */
+
+function onblur_lookupFipsCountry(fipsCountryCode) {
+  singleKeyLookup(
```

Comment:
"singleKeyLookup" appears to be a function introduced by some of KualiCo's CG module scripts, unless you've found it in one of the other third party JavaScript libraries. If you want to use that function, you may need to copy it into this file or import the related CG JavaScript file.

For an ES6-friendly example of invoking a DWR-exposed service, see the "cu-actionlist-common.js" file, which calls an ActionListService method and then updates an appropriate label/message based on the response.

Reply:
Alternatively, for an example of how KFS handles auto-populating certain fields/messages on Person form fields, see the "objectInfo.js" file's loadUserInfo() function.

Reply:
Still looking at this item.